test/functional/feature_fee_estimation.py

```diff
@@ -115,13 +115,13 @@ def check_estimates(node, fees_seen):
     check_smart_estimates(node, fees_seen)
 
 
-def send_tx(wallet, node, utxo, feerate):
+def send_tx(wallet, node, feerate, utxo=None):
```

Comment:
Prefer just passing `None` as `utxo` explicitly, as appropriate.

Reply:
Feels like a hack that would be better off identifying CPFP from the block contents itself. Possibly even using it for the estimates, if practical. But at least this should be better than nothing.